### ISSUE TITLE
Use external URL when vpn active

### ISSUE THREAD

**Issue by I-hate-2FA** [home-assistant/android#4780]
**Is your feature request related to a problem? Please describe.**
When vpn is on, internal url is obviously not reachable, external URL should be used

**Describe the solution you'd like**
Detect if vpn is on, even if network SSID match one in database, use external URL anyways

**Describe alternatives you've considered, if any**
Wait 60 second until it time out and refresh external url

**Additional context**

**Comment by I-hate-2FA** [comment]
never the less, vpn state is more important then wifi, to use external or internal url should be tggleable, but wifi state should always be ignored

**Comment by decimoseptimo** [comment]
+1 This is useful for when connecting to a 3rd party VPN server, instead of a VPN in your home server.

**Comment by I-hate-2FA** [comment]
seems like with recent update a toggle called vpn active is added, but it always use internal url, a second toggle must be added to always use external url

**Comment by I-hate-2FA** [comment]
yes on windows i also have vpn always active and it can access with internal url just fine because windows can detect where its supposed to go and route automatically android a toggle must be added otherwise its not possible to use the internal url feature

**Comment by decimoseptimo** [comment]
What vpn do you use? does it let passthrough private urls?

**Comment by I-hate-2FA** [comment]
no, it is one click type
